internal/mcheck/mod_test.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"golang.org/x/tools/go/analysis/analysistest"
+)
+
+func TestMain(m *testing.M) {
+	os.Exit(m.Run())
```

Comment:
Why is that necessary ?

Reply:
In fact no any more, that was an attempt to increase coverage by testing the main function